### ISSUE TITLE
Precision is over the maximum defined for this asset.

### ISSUE THREAD

**Issue by ebadta81** [wisespace-io/binance-rs#146]
Hello,

Is there an easy way, to handle f64's for account.market_buy?

I get numbers like this ```qty: 119.96000000000001``` , so no surprise, the server gives me this error message.

As I see now, it is not possible with f64, but market_buy need f64.

I consider to make a Decimal version of your asset.rs.

**Comment by wisespace-io** [comment]
@ebadta81 Yes, I see the problem. In the past, I thought to use a decimal library. It is probably but I never did it.
You can use https://crates.io/crates/rust_decimal ... when I have more time, I can think an integrate to the crate.
